### ISSUE TITLE
Auto-import triggers for reserved keyword exports

### ISSUE THREAD

**Issue by rbuckton** [microsoft/TypeScript#28007]
**TypeScript Version:**  3.1.3

**Search Terms:** auto import keyword

**Code**

```ts
// a.ts
function return_() {}
function throw_() {}
function break_() {}
function continue_() {}
export { return_ as return, throw_ as throw, break_ as break, continue_ as continue };

// b.ts

// typing any of the following keywords in VS Code now auto-imports one of the above exports
return
throw
break
continue
```

**Expected behavior:**
Auto-import should not trigger for reserved words.

**Actual behavior:**
Auto-import triggers any time you write a reserved word with the same name as an export, adding an
import declaration to the top of the file for a keyword that is invalid.

**Comment by fatcerberus** [comment]
This is a TypeScript bug?  I’d have thought this was an issue with VS Code, not TS.

**Comment by rbuckton** [comment]
It could be VS Code, but it depends on whether the auto-import suggestions come from Code or the TypeScript language service.

**Comment by fatcerberus** [comment]
Disclaimer: Not familiar with either codebase, but I would guess the flow goes something like:

1. VSCode asks TSLS which module has an export by some name
2. TSLS returns list of modules with an export by that name (doesn’t matter what the name is since it’s legal to use keywords as export names)
3. VSCode adds the relevant import to the file

So if VSCode is making that request for reserved keywords at all, that sounds like a VSCode bug to me and not actually TSLS’s fault.

Of course there’s always that possibility that I have no clue what I’m talking about and if that’s the case then feel free to pretend I never said anything. :)

**Comment by AlCalzone** [comment]
I'm currently running into this while editing a project that uses `tap`, which exports `true` and `false`.
![grafik](https://user-images.githubusercontent.com/17641229/91725985-c83d2800-eb9f-11ea-9b9d-47535e9ce977.png)

This is incredibly annoying, any way to get this prioritized higher?

**Comment by RyanCavanaugh** [comment]
Closing language service bugs related to the 6.0 implementation. For more information, see #62827
